contracts/transfer/src/wasm/spurious.rs

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use crate::Transfer;
+
+use alloc::vec::Vec;
+use canonical::Store;
+use dusk_bls12_381::BlsScalar;
+use phoenix_core::Note;
+
+impl<S: Store> Transfer<S> {
+    pub fn balance(&self, address: BlsScalar) -> u64 {
+        self.balance
+            .get(&address)
+            .unwrap_or_default()
+            .map(|v| *v)
+            .unwrap_or_default()
```

Comment:
Is `unwrap_or_default` the desired behavior for sure?

Reply:
Yep. The function will return `0` for a non-existent balance